### PR TITLE
[Doc enhancement][TOC placement change][Issue 3922]

### PR DESCRIPTION
**Pull request by Rajeshwari-v**
Please review,

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We do not currently enforce a minimum score for content merged to the default branch. This will change in the future.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/TOC.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/6ea5ff4326b2698d545f460c5d922e6a574e184e/msteams-platform/TOC.yml) | 96 | 5 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/30815a10-f895-4c26-99de-45b6541bcbb8) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We do not currently enforce a minimum score for content merged to the default branch. This will change in the future.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/TOC.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/3534c0e52d22d9ee76bae5167aa64a5d4ad09aea/msteams-platform/TOC.yml) | 96 | 5 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/5fb01732-94c5-4270-ba26-11ccabf7c318) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/concepts/device-capabilities/device-capabilities-overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/3534c0e52d22d9ee76bae5167aa64a5d4ad09aea/msteams-platform/concepts/device-capabilities/device-capabilities-overview.md) | 91 | 5 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/4887b0e7-ca81-4ca8-bea6-b387660a2330) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We do not currently enforce a minimum score for content merged to the default branch. This will change in the future.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/TOC.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/df5be228707c0a0abcf7dc1cd9966188a1cf07b4/msteams-platform/TOC.yml) | 96 | 5 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f455de5f-cb50-43f5-bdc3-6cbffc02e773) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/concepts/device-capabilities/device-capabilities-overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/df5be228707c0a0abcf7dc1cd9966188a1cf07b4/msteams-platform/concepts/device-capabilities/device-capabilities-overview.md) | 91 | 5 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/0082b915-6f80-4c28-b45f-6e5452aec58f) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/samples/integrate-web-apps-overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/df5be228707c0a0abcf7dc1cd9966188a1cf07b4/msteams-platform/samples/integrate-web-apps-overview.md) | 83 | 24 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/424ef399-688d-4904-a6dd-27318754e589) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We do not currently enforce a minimum score for content merged to the default branch. This will change in the future.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/TOC.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/bc2ff139c55be512476210732387f9fb8c8e5298/msteams-platform/TOC.yml) | 96 | 5 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/4df9321f-c6a1-4a4a-bb8c-aaf588b2b27e) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/concepts/device-capabilities/device-capabilities-overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/bc2ff139c55be512476210732387f9fb8c8e5298/msteams-platform/concepts/device-capabilities/device-capabilities-overview.md) | 91 | 5 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/98c10dc1-0465-4e15-970c-b1110622cc06) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/concepts/device-capabilities/people-picker-capability.md](https://github.com/MicrosoftDocs/msteams-docs/blob/bc2ff139c55be512476210732387f9fb8c8e5298/msteams-platform/concepts/device-capabilities/people-picker-capability.md) | 81 | 22 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/058ed499-521c-4094-9e40-f02791d9973c) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/samples/integrate-web-apps-overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/bc2ff139c55be512476210732387f9fb8c8e5298/msteams-platform/samples/integrate-web-apps-overview.md) | 83 | 24 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/ab0e6e6e-35e1-4759-a544-dae395f442ea) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We do not currently enforce a minimum score for content merged to the default branch. This will change in the future.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/TOC.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/162d84cca907b30c9625676b00e7eed421c0690d/msteams-platform/TOC.yml) | 96 | 5 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f659ce94-36a4-4a2e-90cd-e08a098a9118) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/concepts/device-capabilities/device-capabilities-overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/162d84cca907b30c9625676b00e7eed421c0690d/msteams-platform/concepts/device-capabilities/device-capabilities-overview.md) | 91 | 5 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/d318016f-55cc-4643-ab79-4ae7c44b8ffc) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/concepts/device-capabilities/location-capability.md](https://github.com/MicrosoftDocs/msteams-docs/blob/162d84cca907b30c9625676b00e7eed421c0690d/msteams-platform/concepts/device-capabilities/location-capability.md) | 82 | 17 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/810912eb-c8ae-4e81-b526-cf1c432662fa) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/concepts/device-capabilities/mobile-camera-image-permissions.md](https://github.com/MicrosoftDocs/msteams-docs/blob/162d84cca907b30c9625676b00e7eed421c0690d/msteams-platform/concepts/device-capabilities/mobile-camera-image-permissions.md) | 87 | 21 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/d509eb05-2ecb-4c2c-bcab-605bd5e384fd) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/concepts/device-capabilities/native-device-permissions.md](https://github.com/MicrosoftDocs/msteams-docs/blob/162d84cca907b30c9625676b00e7eed421c0690d/msteams-platform/concepts/device-capabilities/native-device-permissions.md) | 89 | 17 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/3e0fc811-e41e-4b32-beaf-db8560626c11) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/concepts/device-capabilities/people-picker-capability.md](https://github.com/MicrosoftDocs/msteams-docs/blob/162d84cca907b30c9625676b00e7eed421c0690d/msteams-platform/concepts/device-capabilities/people-picker-capability.md) | 81 | 22 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/9e677823-d9b0-4708-a432-0e2a8c749d9d) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/concepts/device-capabilities/qr-barcode-scanner-capability.md](https://github.com/MicrosoftDocs/msteams-docs/blob/162d84cca907b30c9625676b00e7eed421c0690d/msteams-platform/concepts/device-capabilities/qr-barcode-scanner-capability.md) | 89 | 16 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/88004add-7b0b-4ef8-ab5e-3d36c022139b) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/samples/integrate-web-apps-overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/162d84cca907b30c9625676b00e7eed421c0690d/msteams-platform/samples/integrate-web-apps-overview.md) | 83 | 24 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/31ae44e4-2417-4edb-bf40-dffccff960ec) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/concepts/device-capabilities/device-capabilities-overview.md [hunk 1]
```diff
@@ -37,5 +37,3 @@ After getting access to device capabilities, use Teams media capability APIs to
 * Record audio through microphone.
 * Share location using [location picker](location-capability.md).
 
-Also, you can integrate the Teams native [people picker control](people-picker-capability.md) that allows users to search and select people in the web app experience.
```
Comment (shuchauh) [inline msteams-platform/concepts/device-capabilities/device-capabilities-overview.md:40]: Can we instead rephrase it to highlight this capability and not remove it completely?
To my understanding the comment was around ToC only, so we need not remove the cross referencing part within the document itself.

File: msteams-platform/concepts/device-capabilities/people-picker-capability.md [hunk 2]
```diff
@@ -1,31 +1,31 @@
...
-# Integrate People Picker capability 
+# Integrate People Picker  
 
 People Picker is a control to search and select people. This is a native capability available in Teams platform. You can integrate Teams native People Picker input control with your web apps. You can select between single or multi selection, and configurations, such as limiting search within a chat, channels, or across the entire organization.
 
-You can use [Microsoft Teams JavaScript client SDK](/javascript/api/overview/msteams-client?view=msteams-client-js-latest&preserve-view=true), which provides `selectPeople` API to integrate the People Picker capability within your web app. 
+You can use [Microsoft Teams JavaScript client SDK](/javascript/api/overview/msteams-client?view=msteams-client-js-latest&preserve-view=true), which provides `selectPeople` API to integrate the People Picker within your web app. 
 
-## Advantages of integrating People Picker capability
+## Advantages of integrating People Picker
```
Comment (shuchauh) [inline msteams-platform/concepts/device-capabilities/people-picker-capability.md:17]:
```suggestion
## Advantages of integrating the native People Picker 
```

File: msteams-platform/concepts/device-capabilities/people-picker-capability.md [hunk 3]
```diff
@@ -1,31 +1,31 @@
...
+You can use [Microsoft Teams JavaScript client SDK](/javascript/api/overview/msteams-client?view=msteams-client-js-latest&preserve-view=true), which provides `selectPeople` API to integrate the People Picker within your web app. 
 
-## Advantages of integrating People Picker capability
+## Advantages of integrating People Picker 
 
-* The People Picker control works in all of Teams surfaces, such as task module, a chat, channel, meeting tab, and personal app.
+* People Picker control works in all of Teams surfaces, such as task module, a chat, channel, meeting tab, and personal app.
 * This control allows you to search for and select users within a chat, channel, or the entire organization.
-*  The People Picker capability helps with scenarios involving task assignment, tagging, notifying a user. 
+* People Picker helps with scenarios involving task assignment, tagging, notifying a user.
```
Comment (shuchauh) [inline msteams-platform/concepts/device-capabilities/people-picker-capability.md:21]:
```suggestion
* People Picker control helps with scenarios involving task assignment, tagging, notifying a user. 
```

File: msteams-platform/concepts/device-capabilities/people-picker-capability.md [hunk 4]
```diff
@@ -1,31 +1,31 @@
...
-*  The People Picker capability helps with scenarios involving task assignment, tagging, notifying a user. 
+* People Picker helps with scenarios involving task assignment, tagging, notifying a user. 
 * You can use this readily available control in your web app. It saves the effort and time significantly to build such a control on your own.
 
 You must call the `selectPeople` API to integrate People Picker control in your Teams app. For effective integration, you must have an understanding of [code snippet](#code-snippet) for calling the API. 
 It is important to familiarize yourself with the [API response errors](#error-handling) to handle the errors in your web app.
 
 > [!NOTE] 
-> Currently, Microsoft Teams support for People Picker capability is available for mobile clients only.
+> Currently, Microsoft Teams support for People Picker is available for mobile clients only.
```
Comment (shuchauh) [inline msteams-platform/concepts/device-capabilities/people-picker-capability.md:28]:
```suggestion
> Currently, Microsoft Teams support for the People Picker control is available for mobile clients only.
```

File: msteams-platform/concepts/device-capabilities/people-picker-capability.md [hunk 5]
```diff
@@ -45,9 +45,9 @@ The `selectPeople` API comes along with following input configurations:
 |`openOrgWideSearchInChatOrChannel`|Boolean | It is an optional parameter. When it is set to true, it launches the People Picker in organization wide scope even if the app is added to a chat or channel. |False|
 |`singleSelect`|Boolean|It is an optional parameter. When it is set to true, it launches the People Picker restricting the selection to one user only. |False|
 
-The following image depicts the experience of People Picker capability in a sample web app:
+The following image depicts the experience of People Picker in a sample web app:
```
Comment (shuchauh) [inline msteams-platform/concepts/device-capabilities/people-picker-capability.md:48]:
```suggestion
The following image depicts the experience of the People Picker control in a sample web app:
```

File: msteams-platform/concepts/device-capabilities/people-picker-capability.md [hunk 6]
```diff
@@ -45,9 +45,9 @@ The `selectPeople` API comes along with following input configurations:
...
 
-![Web app experience of People Picker capability](../../assets/images/tabs/people-picker-control-capability.png)
+![Web app experience of People Picker](../../assets/images/tabs/people-picker-control-capability.png)
```
Comment (shuchauh) [inline msteams-platform/concepts/device-capabilities/people-picker-capability.md:50]:
```suggestion
![Web app experience of the People Picker control](../../assets/images/tabs/people-picker-control-capability.png)
```

File: msteams-platform/samples/integrate-web-apps-overview.md [hunk 7]
```diff
@@ -68,9 +68,9 @@ You can get a rich integration experience between Microsoft Teams and SharePoint
 
 You can create deep links to the entities in Teams. You can create links to information and features within Teams. These deep links navigate to content and information within your tab. You can use deep links to link your app with Teams as they tie together multiple pieces of an app for a more native Teams experience.
 
-## Integrate device capabilities
+## Integrate People Picker
 
-Microsoft Teams platform is continuously enhancing developer capabilities aligning with built-in first-party experiences. The enhanced Teams platform allows partners to access and integrate the native device capabilities, such as camera, QR or barcode scanner, photo gallery, microphone, and location using dedicated APIs available in Microsoft Teams JavaScript client SDK. 
+You can integrate the Teams native [people picker control](~/concepts/device-capabilities/people-picker-capability.md) that allows users to search and select people in the web app experience.
```
Comment (shuchauh) [inline msteams-platform/samples/integrate-web-apps-overview.md:73]: Let's keep both device capabilities and People Picker

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We do not currently enforce a minimum score for content merged to the default branch. This will change in the future.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/TOC.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/043d0a6576d0e5e1d3f614b2d30c6142ddfd3016/msteams-platform/TOC.yml) | 96 | 5 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f5f259b3-421d-46a0-8db6-a70bf618422c) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/concepts/device-capabilities/device-capabilities-overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/043d0a6576d0e5e1d3f614b2d30c6142ddfd3016/msteams-platform/concepts/device-capabilities/device-capabilities-overview.md) | 91 | 5 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/088a7f15-625e-48d0-aecf-160266872d2b) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/concepts/device-capabilities/location-capability.md](https://github.com/MicrosoftDocs/msteams-docs/blob/043d0a6576d0e5e1d3f614b2d30c6142ddfd3016/msteams-platform/concepts/device-capabilities/location-capability.md) | 82 | 17 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/50c34bbf-90fa-42be-91c1-c660714e9447) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/concepts/device-capabilities/mobile-camera-image-permissions.md](https://github.com/MicrosoftDocs/msteams-docs/blob/043d0a6576d0e5e1d3f614b2d30c6142ddfd3016/msteams-platform/concepts/device-capabilities/mobile-camera-image-permissions.md) | 87 | 21 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/dd48de1a-0614-4daa-ad64-4a7401cef8fe) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/concepts/device-capabilities/native-device-permissions.md](https://github.com/MicrosoftDocs/msteams-docs/blob/043d0a6576d0e5e1d3f614b2d30c6142ddfd3016/msteams-platform/concepts/device-capabilities/native-device-permissions.md) | 89 | 17 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/41a2c7d5-61d1-4879-bf4c-4141dc3fb6ab) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/concepts/device-capabilities/people-picker-capability.md](https://github.com/MicrosoftDocs/msteams-docs/blob/043d0a6576d0e5e1d3f614b2d30c6142ddfd3016/msteams-platform/concepts/device-capabilities/people-picker-capability.md) | 81 | 22 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/2bb9a097-57df-4ede-985e-22fe6bc2e2e7) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/concepts/device-capabilities/qr-barcode-scanner-capability.md](https://github.com/MicrosoftDocs/msteams-docs/blob/043d0a6576d0e5e1d3f614b2d30c6142ddfd3016/msteams-platform/concepts/device-capabilities/qr-barcode-scanner-capability.md) | 89 | 16 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/c923490d-cc87-4b50-b9d2-ac6447bd5b72) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/samples/integrate-web-apps-overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/043d0a6576d0e5e1d3f614b2d30c6142ddfd3016/msteams-platform/samples/integrate-web-apps-overview.md) | 83 | 24 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/a0b11f79-9e4f-4767-a1d0-233751f6e521) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/concepts/device-capabilities/people-picker-capability.md [hunk 8]
```diff
@@ -1,31 +1,31 @@
...
 ms.author: surbhigupta
 ---
 
-# Integrate People Picker capability 
+# Integrate People Picker  
 
 People Picker is a control to search and select people. This is a native capability available in Teams platform. You can integrate Teams native People Picker input control with your web apps. You can select between single or multi selection, and configurations, such as limiting search within a chat, channels, or across the entire organization.
 
-You can use [Microsoft Teams JavaScript client SDK](/javascript/api/overview/msteams-client?view=msteams-client-js-latest&preserve-view=true), which provides `selectPeople` API to integrate the People Picker capability within your web app. 
+You can use [Microsoft Teams JavaScript client SDK](/javascript/api/overview/msteams-client?view=msteams-client-js-latest&preserve-view=true), which provides `selectPeople` API to integrate the People Picker within your web app.
```
[the remark this answers is not in the document]
Comment (Rajeshwari-v) [reply msteams-platform/concepts/device-capabilities/people-picker-capability.md:15]: Have used People Picker across the docs for consistency.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We do not currently enforce a minimum score for content merged to the default branch. This will change in the future.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/TOC.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/053c25b92ba07a392e751a1876622b93c416c341/msteams-platform/TOC.yml) | 96 | 5 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/80ce4f89-ec22-4dd4-b096-9c809c1dc0db) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/concepts/app-fundamentals-overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/053c25b92ba07a392e751a1876622b93c416c341/msteams-platform/concepts/app-fundamentals-overview.md) | 83 | 10 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/0d387813-b9f5-4608-b7af-a38710e79ec0) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/concepts/device-capabilities/device-capabilities-overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/053c25b92ba07a392e751a1876622b93c416c341/msteams-platform/concepts/device-capabilities/device-capabilities-overview.md) | 91 | 5 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/72eaa2db-efc4-4ec9-b5b6-99bb638e98bb) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/concepts/device-capabilities/location-capability.md](https://github.com/MicrosoftDocs/msteams-docs/blob/053c25b92ba07a392e751a1876622b93c416c341/msteams-platform/concepts/device-capabilities/location-capability.md) | 82 | 17 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/dac4ad06-d41a-48a8-abd7-38ff6f2f7f26) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/concepts/device-capabilities/mobile-camera-image-permissions.md](https://github.com/MicrosoftDocs/msteams-docs/blob/053c25b92ba07a392e751a1876622b93c416c341/msteams-platform/concepts/device-capabilities/mobile-camera-image-permissions.md) | 87 | 21 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f900bc45-dd62-4bf0-9cc3-f3c0601dd34b) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/concepts/device-capabilities/native-device-permissions.md](https://github.com/MicrosoftDocs/msteams-docs/blob/053c25b92ba07a392e751a1876622b93c416c341/msteams-platform/concepts/device-capabilities/native-device-permissions.md) | 89 | 17 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/22ed615c-6842-4fb9-9c04-8fb40252a7ce) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/concepts/device-capabilities/people-picker-capability.md](https://github.com/MicrosoftDocs/msteams-docs/blob/053c25b92ba07a392e751a1876622b93c416c341/msteams-platform/concepts/device-capabilities/people-picker-capability.md) | 82 | 22 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/a16ece26-82af-497b-86f0-bf239ebe1e44) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/concepts/device-capabilities/qr-barcode-scanner-capability.md](https://github.com/MicrosoftDocs/msteams-docs/blob/053c25b92ba07a392e751a1876622b93c416c341/msteams-platform/concepts/device-capabilities/qr-barcode-scanner-capability.md) | 89 | 16 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/0fb1ba9d-0d1a-4745-bcaa-59c2c93b1cc2) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/messaging-extensions/how-to/enable-SSO-auth-me.md](https://github.com/MicrosoftDocs/msteams-docs/blob/053c25b92ba07a392e751a1876622b93c416c341/msteams-platform/messaging-extensions/how-to/enable-SSO-auth-me.md) | 87 | 6 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/7406a7d4-2be5-4ff7-a3c9-8115a37322f8) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/samples/integrate-web-apps-overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/053c25b92ba07a392e751a1876622b93c416c341/msteams-platform/samples/integrate-web-apps-overview.md) | 83 | 25 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f12bc4a4-a372-4465-819e-a5243930bb76) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by v-abirade — APPROVED** [review]
Approved
